pkg/kedge/http/lbtransport/body_test.go

```diff
@@ -0,0 +1,113 @@
+package lbtransport
+
+import (
+	"bytes"
+	"io"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestReplayableReader(t *testing.T) {
+	for _, tcase := range []struct {
+		name string
+		src                 io.Reader
+		sequentialReadBytes []int
+		rewindBeforeRead    []bool
+
+		expectedBytes [][]byte
+		expectedErrs  []error
+	}{
+		{
+			name: "wrapped nil, read should return EOF",
```

Comment:
Nit: Given that these names are used as test names we might want to make them adhere to the usual style with a camel-cased sentence as a single word?